.github/workflows/fcos-podman-next-build.yml

```
@@ -17,36 +19,66 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install qemu dependency
+      run: |
+        sudo apt update
+        sudo apt -y install qemu-user-static
+
     - name: Set up wait-for-copr
       run: |
         pip3 install git+https://github.com/packit/wait-for-copr.git@main
 
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Get short SHA from HEAD
+      run: echo "SHORT_SHA=$(git rev-parse --short HEAD)" >> "$GITHUB_ENV"
+      id: short_sha
+
     - name: Wait for successful podman-next build with the latest commit
       run: |
         # TODO: add this in the Containerfile itself or as a --build-arg
-        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $(git rev-parse --short ${{ github.sha }})
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $SHORT_SHA
         echo "podman-next build successful."
 
-    - name: Check out code
-      uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
-
     - name: Build FCOS Image
-      id: build_image
+      id: build_image_multiarch
       # Ref: https://github.com/redhat-actions/buildah-build
       uses: redhat-actions/buildah-build@v2
       with:
-        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
-        tags: ${{env.IMAGE_TAGS }}
-        containerfiles: |
-          ./contrib/podman-next/fcos-podmanimage/Containerfile .
+        image: ${{ env.IMAGE_NAME }}
+        tags: ${{ env.IMAGE_TAGS }}
+        archs: ${{ env.IMAGE_ARCHS }}
+        containerfiles: ./contrib/podman-next/fcos-podmanimage/Containerfile
+
+    - name: Echo Outputs
+      run: |
+        echo "Image: ${{ steps.build_image_multiarch.outputs.image }}"
+        echo "Tags: ${{ steps.build_image_multiarch.outputs.tags }}"
+        echo "Tagged Image: ${{ steps.build_image_multiarch.outputs.image-with-tag }}"
+
+    - name: Check images created
+      run: buildah images | grep '${{ env.IMAGE_NAME }}'
+
+    - name: Check image metadata
+      run: |
+        set -x
+        # COPR_PROJECT envvar is used for the `podman-next` floating tag
+        buildah inspect ${{ steps.build_image_multiarch.outputs.image }}:${{ env.COPR_PROJECT }} | jq ".OCIv1.architecture"
+        buildah inspect ${{ steps.build_image_multiarch.outputs.image }}:${{ env.COPR_PROJECT }} | jq ".Docker.architecture"
+        buildah inspect ${{ steps.build_image_multiarch.outputs.image }}:${{ github.sha }} | jq ".OCIv1.architecture"
+        buildah inspect ${{ steps.build_image_multiarch.outputs.image }}:${{ github.sha }} | jq ".Docker.architecture"
+
+    - name: Run image
+      run: podman run --rm ${{ steps.build_image_multiarch.outputs.image-with-tag }}
```

Glad to see this verification.  Would you consider actually running podman in the container instead of just bash?

```suggestion
      run: podman run --privileged --rm ${{ steps.build_image_multiarch.outputs.image-with-tag }} podman system info
```

---

let me try this in sandbox.

---

this works. Added to PR.